lib/membrane_opus/parser.ex

```diff
@@ -69,18 +78,35 @@ defmodule Membrane.Opus.Parser do
     {[], state}
   end
 
+  defp prepare_state(state, pts) do
+    cond do
+      state.generate_best_effort_timestamps? and state.pts_current == nil ->
+        %{state | pts_current: 0}
+
+      !state.generate_best_effort_timestamps? and state.queue == <<>> ->
+        %{state | pts_current: pts}
+
+      !state.generate_best_effort_timestamps? and state.pts_current != pts ->
+        raise """
+        PTS values are not continuous
+        """
```

Comment:
If `state.queue` is not `<<>>`, input PTS should be equal to PTS of the first OPUS frame as a whole coming from the input payload